frontend/apps/reader/modules/readerpaging.lua

```diff
@@ -560,14 +562,17 @@ function ReaderPaging:onGotoPercent(percent)
     return true
 end
 
-function ReaderPaging:onGotoViewRel(diff)
+function ReaderPaging:onGotoViewRel(diff, no_page_turn)
+    -- ReaderSearch calls with no_page_turn = true.
+    -- In that case the returned 'ret' is nil when page turning should be done. Actual page turning is not done.
```

Comment:
should be done (sounds like it's something we should do) => would happen
May be:
`In that case, don't turn page if it would happen, and return ret=nil`